.github/workflows/release.yml

```diff
@@ -15,7 +15,9 @@ jobs:
         run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
 
       - name: Build the plugins
-        run: make plugins
+        run: |
+          swift build --build-path=./.build --configuration=release --product protoc-gen-swift
+          swift build --build-path=./.build --configuration=release --product protoc-gen-grpc-swift
```

Comment:
I don't think this'll work, the zipped plugins are looking in the current directory, I think you need to copy them out of the build directory into the current dir. Also I don't think you need to specify the `--built-path` here.

Reply:
Ah, good catch. Fixed.